### PR TITLE
Update circulation web for ThePalaceProject repo

### PR DESCRIPTION
**Pull request by jonathangreen**
## Description

Updates for our fork of the circulation-web repo:
- Update name to be circulation-admin
- Break Github actions into 3 components:
  - Build Docs - Publish github pages docs
  - Test - Test any PR or branch
  - Release - Release to NPM on new tag
- Remove old NPM config

### CODE REVIEW

File: .github/workflows/test-and-deploy.yml [hunk 1]
```diff
@@ -17,6 +17,11 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Install Node.js 💻
+        uses: actions/setup-node@v2
+        with:
+          node-version: 10
```
Comment (tdilauro) [inline .github/workflows/test-and-deploy.yml:21]: Node 10 is [EOL](https://nodejs.org/en/about/releases/). Any reason not to start with at least 12 or, better yet, 14?

File: package.json [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+    "url": "git://github.com/ThePalaceProject/circulation-web.git"
   },
   "main": "dist/circulation-web.js",
-  "author": "NYPL",
+  "author": "The Palace Project",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=8.0.0",
-    "npm": ">=5.0.0"
+    "node": ">=10.0.0",
```
Comment (tdilauro) [inline package.json:12]: Same node 10 EOL issue here.

**Review by tdilauro — COMMENTED** [review]
LGTM. Especially like splitting up the GH Actions workflows.

**Comment by jonathangreen** [comment]
Updated to use Node 12 everywhere.

**Pull request merged**